debian/rules

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE = 1
+SERVICE = neofs-node
+VERSION = $(shell make --no-print-directory version | sed "s/^v//" | sed -r "s/(.*)-(g[a-fA-F0-9]{6,8})(.*)/\1\3~\2/" | sed "s/-/~/")
+DISTRIBUTION = $(shell sed -n "s/^VERSION_CODENAME=//p" /etc/os-release)
+PACKAGEVERSION = $(VERSION)-$(DISTRIBUTION)
+
+%:
+	dh $@ --with sysuser,bash-completion
+
+override_dh_auto_test:
+
+override_dh_auto_install:
+		dh_auto_install
+		install -D -m 0750 -t debian/$(SERVICE)/bin bin/*
+		bin/neofs-adm gendoc -t man man/
+		bin/neofs-adm completion bash > debian/neofs-adm.bash-completion
+		bin/neofs-cli gendoc -t man man/
+		bin/neofs-cli completion bash > debian/neofs-cli.bash-completion
+		install -m 0755 -d debian/$(SERVICE)/usr/share/fish/completions
+		bin/neofs-cli completion fish > debian/$(SERVICE)/usr/share/fish/completions/neofs-node.fish
+		install -m 0755 -d debian/$(SERVICE)/usr/share/zsh/vendor-completions
+		bin/neofs-cli completion zsh > debian/$(SERVICE)/usr/share/zsh/vendor-completions/_neofs-node
+		wget https://github.com/nspcc-dev/neofs-locode-db/releases/download/v0.2.1/locode_db.tar.gz
+		install -d debian/$(SERVICE)/var/lib/neofs/ir
+		tar xvf locode_db.tar.gz -C debian/$(SERVICE)/var/lib/neofs/ir/
+		install -T config/example/ir.yaml debian/$(SERVICE)/etc/neofs/ir/config.yml
+		install -T config/example/ir-control.yaml debian/$(SERVICE)/etc/neofs/ir/control.yml
+		install -T config/example/node.yaml debian/$(SERVICE)/etc/neofs/storage/config.yml
+		install -T config/example/node-control.yaml debian/$(SERVICE)/etc/neofs/storage/control.yml
+
+
+override_dh_installsystemd:
+		dh_installsystemd --no-enable --no-start --name=neofs-ir
+		dh_installsystemd --no-enable --no-start --name=neofs-storage
+
+override_dh_installchangelogs:
+		dh_installchangelogs -k CHANGELOG.md
```

Comment:
Do you really need it? It should be handled automatically with proper changelog/changelog.Debian separation.

Reply:
It's the most simple way of separation AFAIK.
Could you please suggest the other one?

Reply:
Drop the override and see if it works?

Reply:
well the original "CHANGELOG.md" has been renamed to "changelog"

Reply:
And we can forget about this override which seems to be worthwhile to me. Not critical, but why bother with overrides if everything works already by default?

Reply:
It does not.
Origin file is renamed, file extension is lost - could break some visualisation I think.

Reply:
Let's keep the override with .md